examples/tensorflow/distribution_strategy/estimator-API/Dockerfile

```diff
@@ -1,4 +0,0 @@
-FROM tensorflow/tensorflow:1.11.0
```

Comment:
Why did you remove the estimator example ?

Reply:
As we discussed, this is the example that was updated to TF-V2 with [new example](https://github.com/kubeflow/training-operator/tree/master/examples/tensorflow/distribution_strategy/keras-API) that uses TF-v2 and keras API (instead of estimator API - which is more relevant from estimator API. BTW, ASFAIK estimator not available on TF for version 2.16+)